### PR TITLE
Exercise 1

### PR DESCRIPTION
**Pull request by kogoe1**

### CODE REVIEW

File: todo_app/app.py [hunk 1]
```diff
@@ -1,15 +1,40 @@
...
+
 
 app = Flask(__name__)
 app.config.from_object(Config)
 
+def taskSort(item):
+    return item['status']
 
-@app.route('/')
+@app.route('/', methods=['GET', 'POST'])
```
[the remark this answers is not in the document]
Comment (kogoe1) [reply todo_app/app.py]: This has been changed as suggested

File: todo_app/data/forms.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+from flask_wtf import FlaskForm
```
[the remark this answers is not in the document]
Comment (kogoe1) [reply todo_app/data/forms.py:1]: This is also now sorted

File: todo_app/app.py [hunk 3]
```diff
@@ -1,15 +1,40 @@
-from flask import Flask
 
+from flask import Flask, render_template, redirect, request
 from todo_app.flask_config import Config
+from todo_app.data.forms import *
```
[the remark this answers is not in the document]
Comment (kogoe1) [reply todo_app/app.py]: Imports have been tidied up

File: todo_app/flask_config.py [hunk 4]
```diff
@@ -3,6 +3,6 @@
 
 class Config:
     """Base configuration variables."""
-    SECRET_KEY = os.environ.get('SECRET_KEY')
+    SECRET_KEY = os.environ.get('SECRET_KEY') or "you-will-never-guess-this"
```
[the remark this answers is not in the document]
Comment (kogoe1) [reply todo_app/flask_config.py]: Change has been reverted and a local .env file has been created with appropriate config values

File: todo_app/templates/index.html [hunk 5]
```diff
@@ -7,11 +7,51 @@ <h1 class="display-4">To-Do App</h1>
...
+  <div class="row">
+    <div class="col-md-12">
+      <h2>Add New TODO Item</h2>
+      <form action="" method="post" novalidate>
+        {{ form.hidden_tag() }}
+        <p>
+            {{ form.title.label }}<br>
+            {{ form.title(size=60) }}
+        </p>
+        <p>{{ form.submit() }}</p>
```
[the remark this answers is not in the document]
Comment (kogoe1) [reply todo_app/templates/index.html:19]: Shall have this added and push the changes once done

**Review by Jonesey13 — APPROVED** [review]
Thanks for making the changes suggested. All looks good to me now 👍

**Pull request merged**